.azure-pipelines/build_and_install_module.sh

```diff
@@ -66,6 +66,8 @@ function build_and_install_kmodule()
         cp drivers/net/team/*.ko $TEAM_DIR/
         modinfo $TEAM_DIR/team.ko
         depmod
+        echo 'dmesg'
+        dmesg
         modprobe team
```

Comment:
`set -e` is present at the top of the file, so it won't execute anything after this command fails. I think `modprobe team || true` immediately followed by `dmesg` would help.

Reply:
sure.. let me add that as well, meantime you can check if the current one has any clue